origin-growth/migrations/20190226142347-update-growth-campaign.js

```diff
@@ -0,0 +1,110 @@
+'use strict'
```

Comment:
No need to create a new migration file. We haven't run those growth migrations yet on staging/prod.
So feel free to just modify the existing migration file that creates that table.

Reply:
aha, tnx updating the initial migration